aten/src/ATen/native/vulkan/VulkanOps.cpp

```diff
@@ -218,13 +220,17 @@ void conv2d_depthwise(
     int32_t dilate[2];
     int32_t inputSize[4];
     int32_t outputSize[4];
+    float outputMin;
+    float outputMax;
   };
   ConstBlock cb{{params.PX, params.PY},
                 {params.KW, params.KH},
                 {params.SX, params.SY},
                 {params.DX, params.DY},
                 {params.OW, params.OH, params.OC_4, 0},
-                {params.W, params.H, params.C_4, 0}};
+                {params.W, params.H, params.C_4, 0},
+                output_min ? *output_min : std::numeric_limits<float>::min(),
+                output_max ? *output_max : std::numeric_limits<float>::max()};
```

Comment:
These should be -std::numeric_limits<float>::infinity() and +std::numeric_limits<float>::infinity() respectively.

std::numeric_limits<float>::min() returns FLT_MIN which is the minimum normalized positive value of float.  Pretty confusing naming.  std::numeric_limits<float>::max() returns FLT_MAX which represents the maximum finite value of float - okay in practice but infinity is preferable in theory.

Considering that all GPUs (I am aware of) implement the IEEE 754 floating-point standard, they should understand these bit representations.